server/requirements.txt

```diff
@@ -1,3 +1,4 @@
+cefpython3>=66
```

Comment:
is there a way to make these requirements optional (eg, put in extras)?   Does not seem appropriate to make them hard requirements given the experimental/prototype nature of this PR.

Reply:
yes, that was an oversight on my part. Will absolutely fix.